### ISSUE TITLE
Pretrained U-NET Parse Error in Digits 6.1

### ISSUE THREAD

**Issue by nigelparsad** [NVIDIA/DIGITS#1991]
I am trying to run the U-NET semantic segmentation pretrained model on some test data to get a feel for the model.   This is my first experiment with this model.  Whether I run the model from a DIGITS docker image or from the DIGITS digits-devserver I built from source, I get the same error:

ParseError
1:1 : Message type "caffe.NetParameter" has no field named "from".

Here are my NGC DIGITS docker images:

$ docker images

REPOSITORY                  TAG                 IMAGE ID            CREATED             SIZE
nvidia/digits                       latest               32ba009273c8     7 days ago           2.8GB
nvcr.io/nvidia/digits           18.02               4f7dc6bf79a6       3 weeks ago         5.45GB

FYI I am using the /nvidia/digits image as the nvcr.io/nvidia/digits images runs for a few seconds then exits gracefully without any error messages generated in the terminal (I run it with the interactive (-it) flag).  That issue will be the topic of another ticket.

Here are some details from the failed runs:

TRAINING DATA:

A.  FEATURES  Folder:

---> 30 single channel 572x572 grayscale PNG images.  Each PNGis a single channel, 8-bit   
      grayscale image 

---> Images are named from 01.png to 30.png

---> Each image file's properties were verfied via file and identify on the command line, e.g.:

      $ file 01.png
      01.png: PNG image data, 572 x 572, 8-bit grayscale, non-interlaced

B.  LABELS Folder:

---> Same PNG image specifications and naming convention as above

---> Each image is a binary mask with white (255) labeling ground truth segmentation and black (0) labeling background

DATASET & SEGMENTATION MODEL CREATION:

C.  NEW DATASET:

![dataset](https://user-images.githubusercontent.com/36293682/37232256-ddd0c8ae-23b3-11e8-8f13-fe234bd1a19a.jpg)

D.  NEW SEGMENTATION MODEL:

![model](https://user-images.githubusercontent.com/36293682/37232436-a1007dec-23b4-11e8-8fed-a37ddb9d7964.jpg)

E.  ERROR:

![error](https://user-images.githubusercontent.com/36293682/37232843-4ad5ed7e-23b6-11e8-8a16-9d428f487427.jpg)

All comments/critiques are appreciated.

**Comment by MiralPatel3011** [comment]
i have same issue. how you resolve it. Thank you
